test/integration/ae_mdw_web/controllers/name_controller_test.exs

```diff
@@ -295,10 +295,13 @@ defmodule Integration.AeMdwWeb.NameControllerTest do
       assert %{"data" => auctions} =
                conn |> get("/names/auctions", limit: limit) |> json_response(200)
 
-      plain_names = Enum.map(auctions, fn %{"name" => plain_name} -> plain_name end)
+      plain_names =
+        Enum.map(auctions, fn %{"name" => plain_name, "info" => %{"auction_end" => auction_end}} ->
+          {plain_name, auction_end}
+        end)
 
       assert length(auctions) <= limit
-      assert ^plain_names = Enum.sort(plain_names)
+      assert ^plain_names = Enum.sort_by(plain_names, fn {_name, expire} -> expire end, :desc)
```

Comment:
If we're not going to need the names (only expirations), why add them to the plain_names list?
```elixir
expirations = Enum.map(auctions, fn %{"info" => %{"auction_end" => auction_end}} -> auction_and end)
# ...
assert ^expirations = Enum.sort(expirations)
```

Reply:
added a minimum name validation